### DISCUSSION TITLE
챗봇 기사자료(PPT 및 기반 정보 확인)

### DISCUSSION THREAD

**Discussion by InChil2** [InChil2/SlotTagging#25] [참고자료]
[기존에 있는 사이렌 오더 효과1](https://moneys.mt.co.kr/news/mwView.php?no=2021050308568034796)

**Comment by InChil2** [comment]
[사이렌오더 실적](https://www.fnnews.com/news/202105161740216991)

**Comment by hharimjung** [comment]
[스타벅스 AI바리스타](https://biz.chosun.com/site/data/html_dir/2017/10/24/2017102401353.html)

**Comment by InChil2** [comment]
[사이렌오더 장단점](https://yourpresence.tistory.com/203)
